cirq/contrib/quimb/mps_simulator.py

```diff
@@ -282,12 +285,15 @@ def sample(
 
 
 @value.value_equality
-class MPSState:
+class MPSState(ActOnArgs):
     """A state of the MPS simulation."""
 
     def __init__(
         self,
         qubit_map: Dict['cirq.Qid', int],
+        axes: Iterable[int],
+        prng: np.random.RandomState,
+        log_of_measurement_results: Dict[str, Any],
```

Comment:
These can probably be made optional (with the possible exception of `prng`), with default values assigned in the constructor body.